google/resource_storage_bucket_object_test.go

```diff
@@ -44,6 +45,45 @@ func TestAccGoogleStorageObject_basic(t *testing.T) {
 	})
 }
 
+func TestAccGoogleStorageObject_update(t *testing.T) {
```

Comment:
I don't think we should use the `_update` suffix here. Those tests in general test the `update` method in the resource module, whereas the parameter we are updating forces a new resource.

Reply:
You are right. Do you think a `_recreate` suffix would be fine?